pages.pt_BR/windows/robocopy.md

```diff
@@ -0,0 +1,26 @@
+# Robocopy
+
+> Cópia robusta de arquivos e pastas.
+> Por padrão, os arquivos só serão copiados se a origem e o destino tiverem diferentes carimbos de data/hora ou tamanhos de arquivo diferentes.
```

Comment:
Here we can leave `timestamp` because the usage of the expression `carimbo de data/hora` is very rare, if not nonexistent, by Brazilian Portuguese speakers.

```suggestion
> Por padrão, os arquivos só serão copiados se a origem e o destino tiverem timestamps ou tamanhos de arquivo diferentes.
```